knexfile.js

```diff
@@ -8,20 +8,20 @@ require('dotenv').config();
 module.exports = {
 
   development: {
-    client: process.env.CLIENT || 'pg',
```

Comment:
This client requires a package to be installed so we can be more firm about what envs will use it. If they changed this with the env var, but then didn't load the package, it would break.